ios/PurchasesHybridCommon/PurchasesHybridCommon/EntitlementVerificationMode+HybridAdditions.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  EntitlementVerificationMode+HybridAdditions.swift
+//  PurchasesHybridCommon
+//
+//  Created by Nacho Soto on 6/27/23.
+//
+
+import RevenueCat
+
+extension Configuration.EntitlementVerificationMode {
+
+    var name: String {
+        switch self {
+        case .disabled: return "DISABLED"
+        case .informational: return "INFORMATIONAL"
+        case .enforced: return "ENFORCED"
+        }
+    }
+
+    init?(name: String) {
+        if let mode = Self.modesByName[name] {
+            self = mode
+        } else {
+            return nil
+        }
+    }
+
+    private static let modesByName: [String: Self] = Dictionary(uniqueKeysWithValues: [
+        Self.disabled,
+        Self.informational,
+        // Disabled temporarily since enforced is not available yet.
+        // Self.enforced
```

Comment:
I've disabled this for now... We can reenable later once we've actually shipped versions supporting this.